scripts/common.py

```diff
@@ -165,6 +188,104 @@ def ensure_release_scripts_folder_exists_and_is_up_to_date():
         subprocess.call(['git', 'pull', remote_alias])
 
 
+def is_port_open(port):
+    """Checks if a process is listening to the port.
+
+    Args:
+        port: int. The port number.
+
+    Return:
+        bool. True if port is open else False.
+    """
+    with contextlib.closing(
+        socket.socket(socket.AF_INET, socket.SOCK_STREAM)) as s:
+        return bool(not s.connect_ex(('localhost', port)))
+
+
+def kill_process(port):
+    """Kills a process that is listening to a specific port.
+    Credits: https://stackoverflow.com/a/20691431/11755830
+
+    Args:
+        port: int. The port number.
+    """
+    for process in psutil.process_iter():
+        for conns in process.connections(kind='inet'):
+            if conns.laddr.port == port:
+                process.send_signal(signal.SIGTERM)
+
+
+def run_command(command):
```

Comment:
Let's delete this function, and just pass stuff to subprocess directly using lists.

Reply:
Done